main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
   "github.com/hashicorp/terraform/plugin"
   "github.com/hashicorp/terraform/terraform"
-  "github.com/Mastercard/terraform-provider-restapi/restapi"
+  "github.com/TMaYaD/terraform-provider-restapi/restapi"
```

Comment:
This will need to be fixed before merging

Reply:
Done.